bin/main.rb

```diff
@@ -0,0 +1,44 @@
+puts "Welcome to Tic Tac Toe!"
+puts "-----------------------"
+
+puts "Play a game with (1) Computer, (2) Your Friend:"
+game_mode = gets.chomp
+
+puts "You selected #{game_mode}"
+
+if game_mode == 1
+    puts "Sorry AGI has not been invented yet, check back in 2020!"
+elsif game_mode == 2
+    puts "Enter the name of Player 1:"
+    player1 = gets.chomp
+
+    puts "Enter the name of Player 2:"
+    player2 = gets.chomp
+else
+    puts "Please select (1) or (2)"
+end
```

Comment:
It will always display this
```ruby
Play a game with (1) Computer, (2) Your Friend:
2
You selected 2
Please select (1) or (2)
Select (X) or (O)
```

It will never ask **Player names**.
You need to convert the read `game_mode` to integer
```ruby
game_mode = gets.chomp.to_i
```

Reply:
Thank you for your feedback, will make the file executable and resubmit. :+1: